app/src/main/res/values-nb-rNO/strings_settings.xml

```diff
@@ -165,7 +165,7 @@
     <string name="set_immersive_mode">Oppslukende modus</string>
     <string name="set_immersive_mode_help">Skjul systembjelker (lang-trykk på hjemmeskjermen for å påtvinge)</string>
     <string name="set_interactive_clock">Interaktiv klokke</string>
-    <string name="set_interactive_clock_help">Klikk på datoen for å åpne forvalgt kalenderprogram, og klokkn for å åpne valgt klokkeprogram.</string>
+    <string name="set_interactive_clock_help">Klikk på datoen for å åpne forvalgt kalenderprogram, og klokken for å åpne valgt klokkeprogram.</string>

```

Comment:
I have moved this change in PR #277 so that it is not blocked while we are discussing the strings rework.

Reply:
I will try to commit a rollback, maybe this would work.
```suggestion
    <string name="set_interactive_clock_help">Klikk på datoen for å åpne forvalgt kalenderprogram, og klokkn for å åpne valgt klokkeprogram.</string>
```

Reply:
Yes it worked, so this change has been integrated in PR #277.